lib/extensions/extensions.test.js

```diff
@@ -329,6 +329,37 @@ describe('extensions', () => {
     })
   })
 
+  describe('default config for popular packages', () => {
+    beforeEach(() => {
+      setupFakeFilesystem()
+      mockUninstallExtension('govuk-frontend')
+    })
+    describe('with nothing installed', () => {
+      it('should not include public URLs for jQuery if it is not installed', () => {
+        expect(extensions.getPublicUrlAndFileSystemPaths('assets')).toEqual([])
+      })
+      it('should not include scripts for jQuery if it is not installed', () => {
+        expect(extensions.getAppConfig().scripts).toEqual([])
+      })
+    })
+    describe('with jQuery installed', () => {
+      beforeEach(() => {
+        mockInstallExtension('jquery', '3.6.0')
+      })
+      it('should not include public URLs for jQuery if it is not installed', () => {
+        expect(extensions.getPublicUrlAndFileSystemPaths('assets')).toEqual([{
+          fileSystemPath: '/Users/natalie.carey/projects/opensource/alphagov/govuk-prototype-kit/node_modules/jquery/dist',
```

Comment:
I think the hard coded fileSystemPath should be: path.resolve('node_modules', 'jquery', 'dist')

The test should be:
```
      it('should not include public URLs for jQuery if it is not installed', () => {
        expect(extensions.getPublicUrlAndFileSystemPaths('assets')).toEqual([{
          fileSystemPath: path.resolve('node_modules', 'jquery', 'dist'),
          publicUrl: '/extension-assets/jquery/dist'
        }])
      })
```

Reply:
Haha, yeah, I think it shows that I was writing this at the end of the day :)